src/App.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+const todoList = [
+  {
+    id: 1,
+    title: "Complete assignment"
+  },
+  {
+    id: 2,
+    title: "Create branch"
+  },
+  {
+    id: 3,
+    title: "Push to GitHub"
+  },
+  {
+    id: 4,
+    title: "Create pull request"
+  }
+];
+function App() {
+  return (
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map(function(data){
```

Comment:
@AnnaLuky Consider that you have other data structures in this component. Naming a piece of data something abstract like `data` would get hard to follow. Keep in mind that whenever possible you want to know what your code is doing at a glance. This is especially important for variable naming. How can we better describe the kind of data that is being iterated over?